modules/account-lib/src/coin/dot/utils.ts

```diff
@@ -27,8 +49,130 @@ export class Utils implements BaseUtils {
     throw new NotImplementedError('method not implemented');
   }
 
+  /**
+   * Verifies the signature on a given message
+   *
+   * @param {string} signedMessage the signed message for the signature
+   * @param {string} signature the signature to verify
+   * @param {string} address the address of the signer
+   * @returns {boolean} whether the signature is valid or not
+   */
+  verifySignature(signedMessage: string, signature: string, address: string): boolean {
+    const publicKey = decodeAddress(address);
+    const hexPublicKey = u8aToHex(publicKey);
+
+    return signatureVerify(signedMessage, signature, hexPublicKey).isValid;
+  }
+
   /** @inheritdoc */
```

Comment:
@gibrangul95  Why this and other isValid Utils are not implemented like:
- isValidTransactionId
- isValidPublicKey
- isValidPrivateKey
- isValidBlockId

Reply:
yes, i was looking into core and ended up implmenting these on my branch there, pushing them in here as well. Also, we can't check for valid block ids or transaction ids

Reply:
@rliberal-simtlix pushed
https://github.com/BitGo/BitGoJS/pull/1658/commits/e04270a67882accec2d9d40584ddc6532be8641c

Reply:
Thank you again @gibrangul95.
@argjv We need here to find a way how to do this validations for block and transaction id, of course if it necessary.